docs/bnb_smart_chain/governance/index.md

```diff
@@ -0,0 +1,102 @@
+# BSC Governance Overview
+
+[BEP-297](https://github.com/bnb-chain/BEPs/pull/297) introduces the native governance module for BSC.
+The governance module is derived from [OpenZeppelin Governor](https://docs.openzeppelin.com/contracts/4.x/governance), with the following features:
+
+- Any staking credit holder is allowed to propose and vote.
+- Voters can continue to earn staking rewards during the voting period.
+- Users can delegate someone else to participate in governance on their behalf.
+- A time lock period is introduced before the proposal is executed.
+- If the proposal is rejected, the proposer does not incur any financial losses.
+
+## Workflow
+
+The workflow of the governance module consists of three stages: submit proposal, cast vote, and execute proposal. Each
+stage has its own requirements and parameters, which can be configured by the BNB smart chain governance parameters.
+
+### Submit Proposal
+
+To submit a proposal, a staking credit holder needs to send a `propose` transaction to the `Governor` contract,
+which is a system contract and the address is `0x0000000000000000000000000000000000002004`,
+specifying the following information:
```

Comment:
I think this is an overview introduction, we dont have list each params and explain them, please simplify this part.

Reply:
fixed